### DISCUSSION TITLE
[DOC]插U盘才能启动解决办法

### DISCUSSION THREAD

**Discussion by archfish** [ophub/amlogic-s9xxx-armbian#1420] [Amlogic]
我的设备是CM-311-1a YST，从5.15.108插着一个ext4U盘升级到5.15.112后无法正常启动，重新做了一个引导U盘插上可以顺利启动机器上的系统。

摸索了了一下解决了问题：

1、制作一个armbian U盘，正常引导进机器的系统后拔出U盘；
2、再次升级内核：armbian-update -k 5.15.111
3、自动重启后问题应该就好了；
4、如果还没好，就插入U盘再进入系统，拔掉U盘后执行： armbian-update
5、重启后应该就好了（我的是这样修复的）

如果卡在下载内核步骤很久，可能有两种原因：
1、网络不通，下载不了镜像
2、其他未知问题，我是清理了/tmp目录： rm -rf /tmp/*

大佬看一下是不是插着U盘更新内核的时候执行了什么命令导致引导分区错了，修复前fstab内容没有记录，无法比对

**Comment by Kloping** [comment]
通过u盘启动系统后,拔掉u盘 之后所有命令都失效 并且 在断开ssh连接后无法再次连接

**Reply by 3611895075** [reply]
这里说的应该是写入emmc后插U盘才能启动

**Comment by dodoy6830** [comment]
今天我按楼主的方法试了，也不行。是6.1.66   内核升级到6.1.68  ，还是只能插上U盘才能引导盒子。引导的是emmc

**Reply by justin0x5eed** [reply]
我也遇到同样的问题，型号是cm311-1a

**Reply by justin0x5eed** [reply]
我找到一个解决办法，就是关机后断电10-20分钟，我试了几次，都可以拔掉U盘启动Armbian。

**Reply by dodoy6830** [reply]
我找到的方法就是，不要插网线安装。

**Comment by dodoy6830** [comment]
你多试试，看看能不能找到合适的方法。我找到一个解决方法还在测试。

**Reply by justin0x5eed** [reply]
想问问你是通过什么方法来解决呢？
